packages/admin/cms-admin/src/pages/createEditPageNode.tsx

```diff
@@ -149,7 +149,7 @@ export function createEditPageNode({
                         slug: newSlug,
                         scope,
                     },
-                    fetchPolicy: "network-only",
+                    fetchPolicy: "cache-first",
```

Comment:
`network-only` is problematic because after a failed submission another validation is triggered (I'm not sure why). This leads to a situation where the SaveButton shows a failed submission but the validation error isn't yet shown (only once the second validation request is finished) -> bad UX.

Therefore I switched to `cache-first` and added a `cache.evict()` on initial render to reset the cache.